.github/workflows/publish-code.yml

```diff
@@ -72,3 +72,26 @@ jobs:
       - name: Publish Nuget packages to Nuget registry
         working-directory: src
         run: dotnet nuget push "out/*" -k ${{secrets.NUGET_AUTH_TOKEN}}
+      - name: Zip documentation
+        working-directory: docs2/public
+        run: zip -r ../../src/out/Documentation.zip "./*"
+      - name: Publish release artifacts
+        uses: actions/github-script@v2
```

Comment:
It's a shame that we have to resort to a lower-level tool instead of single call to `upload-release-asset` action.

Reply:
Maybe add `working-directory: src ` ?

Reply:
Tried but that argument is only valid when using runs and not uses

Reply:
Ah, exactly.